benchmark/kernels/fused_moe_triton/benchmark_vllm_vs_sglang_fused_moe_triton.py

```diff
@@ -141,8 +177,13 @@ def benchmark(batch_size, provider, model_config, use_fp8=False):
     shard_intermediate_size = model_config["shard_intermediate_size"]
     topk = model_config["topk"]
     dtype = model_config["dtype"]
+    block_shape = model_config["block_shape"]
 
     x = torch.randn(num_tokens, hidden_size, dtype=dtype)
+    w1_scale = None
```

Comment:
```suggestion
    w1_scale = w2_scale = a1_scale = a2_scale = None
```

and you can delete line 221.